### PR TITLE
Review extension

### PR DESCRIPTION
**Pull request by jpmckinney**
sustainabilityGoal.csv

* [x] Where does the term "formal employment" come from?
* [x] Should innovation be under economic?

sustainabilityStrategy.csv

* [x] "for example, demanding SPP-related tasks as part of the contract's implementation" is not a specific example. Can we replace it with one?

### CODE REVIEW

**Comment by yolile** [comment]
> Where does the term "formal employment" come from?

From the toolkit https://sustainable.open-contracting.org/what-is-spp-and-open-spp (see the image)

**Comment by yolile** [comment]
> Should innovation be under economic?

I understood we agreed to keeping it separate as per https://github.com/open-contracting/standard/issues/1543#issuecomment-1218525523

**Comment by jpmckinney** [comment]
Hmm, well I had written:

> However, I think we should try to stick to the sustainability/social objectives/innovation framing rather than forcing a mapping to environmental/social/ economic, because the relationship I described above might not always hold

But since the SPP guide uses environmental/social/economic, we should follow that, in which innovation is part of economic. Otherwise, we end up mixing the two different approaches to conceptual hierarchy.

**Comment by yolile** [comment]
> "for example, demanding SPP-related tasks as part of the contract's implementation" is not a specific example. Can we replace it with one?

From the [toolkit](https://sustainable.open-contracting.org/implement/prepare-contract-obligations), one example could be "For example, requiring suppliers continuous monitoring of sustainability-related indicators, such as carbon emissions during the contract. "

**Comment by jpmckinney** [comment]
All done - just need to decide on 'innovation'

**Comment by yolile** [comment]
> All done - just need to decide on 'innovation'

I guess the options are, (1) remove it from the codelist completely, (2) add economic.marketInnovationPromotion, (3) add _social.innovationSomething,_ or (2) and (3)

**Comment by jpmckinney** [comment]
Let's do (2) for now and see if there is evidence for (3) over time.

**Comment by yolile** [comment]
Done

File: codelists/sustainabilityGoal.csv [hunk 1]
```diff
@@ -1,15 +1,15 @@
...
-environmental.wasteReduction,Environmental waste reduction,The goal is to reduce waste.
-innovation,Innovation,The goal is related to promoting innovation to achieve economic growth or efficient resources use.
-social,Social,"The goal is related to generating benefits for society, such as ensuring labor rights, promoting diversity and equal opportunities, and occupational health and safety."
-social.laborRightsPromotion,Social labor rights promotion,The goal is to promote labor rights.
-social.smesInclusion,Social SMEs inclusion,The goal is to promote Small and Medium-sized enterprises (SMEs) participation.
-social.womanInclusion,Social woman inclusion,The goal is to promote women-owned suppliers' participation.
+economic,Economic,"The goal is economic, such as formal employment and the local economy."
+economic.formalEmploymentPromotion,Formal employment promotion,The goal is to promote formal employment.
+economic.localEconomyPromotion,Local economy promotion,The goal is to promote the local economy.
+economic.marketInnovationPromotion,Market innovation promotion,The goal is to promote the market innovation.
```
Comment (jpmckinney) [inline codelists/sustainabilityGoal.csv]:
```suggestion
economic.marketInnovationPromotion,Market innovation promotion,The goal is to promote market innovation.
```

**Review by jpmckinney — COMMENTED** [review]
Looks good, just one edit before merge.

**Comment by yolile** [comment]
@jpmckinney I've added `reservedParticipation` in https://github.com/open-contracting-extensions/ocds_sustainability_extension/pull/2/commits/8a125bdadcd934841a45272ff83db3d46a54281c

**Comment by jpmckinney** [comment]
Ready to merge – just fixed the pluralization.

**Pull request merged**